client/Brocfile.js

```diff
@@ -1,11 +1,23 @@
-/* global require, module */
-
+/* global require, module, process */
+var env = process.env.EMBER_ENV;
```

Comment:
Looking back at this code again now (in the light of day) I think we could wait to initialize the `config` variable after we have instantiated the `app` variable and then declare `config` var like so
```
var EmberApp = require('ember-cli/lib/broccoli/ember-app');
var app = new EmberApp({
  hinting: false
});
var config = require('./config/environment')(app.env);
```
The only longer term downside to this approach is that we wouldn't have access to the config when initializing `app` but since we are not using that variable in it's instantiation presently I don't see that as a problem now.